### ISSUE TITLE
[Feature Request] Themed icons in apply icon pack menu / reset icons option.

### ISSUE THREAD

**Issue by Naveed-SNR** [KieronQuinn/PixelLauncherMods#169]
1. Since, the app already lets you apply themed icons for individual apps, can we also have the themed icons option in the apply icon pack menu so that we can apply themed icons to all apps at once like a regular icon pack? 

2. The apps for which I change icons individually do not change when I apply a new icon pack. So, an option to reset only the icons would be nice because it will allow to change icons for those apps without having to change them individually after applying a new icon pack.

**Comment by KieronQuinn** [comment]
- The first point is intended behaviour, the auto-generated themed icons vary wildly in quality so themed icons have to be applied one by one.

- The second point may be possible though, with additional options during resetting

**Comment by Naveed-SNR** [comment]
> - The first point is intended behaviour, the auto-generated themed icons vary wildly in quality so themed icons have to be applied one by one.
> 
> - The second point may be possible though, with additional options during resetting 

Indeed, the auto-generated themed icons vary in quality but the option would still be helpful for users (like me lol) using some moded version of themed icons (magisk modules, lawnicons etc). That said, having the option would be nice anyway.
